asteroid/scripts/asteroid_cli.py

```diff
@@ -140,7 +140,20 @@ def infer():
     parser.add_argument(
         "-o", "--output-dir", default=None, type=str, help="Output directory to save files."
     )
-    args = parser.parse_args()
+    parser.add_argument(
+        "-d",
+        "--device",
+        default=None,
+        type=str,
+        help="Device to run the model on, eg. 'cuda:0'."
+        "Defaults to 'cuda' if CUDA is available, else 'cpu'.",
+    )
+    args = parser.parse_args(argv)
+
+    if args.device is None:
+        device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
Should have a `ifcuda(with_cuda=..., without_cuda=...)` function? 
This could be rewritten as 
```python
device = args.device if args.device is not None else ifcuda("cuda", "cpu") 
```

Reply:
Hmmmm... not convinced :-D because that function would be exactly

```
def ifcuda(a, b):
    return a if torch.cuda.is_available() else b
```

which isn't much different to just inlining that exact code

Reply:
Yes, it would be exactly this. 
Inlining that code would be 
```python
device = args.device if args.device is not None else "cuda" if torch.cuda.is_available() else "cpu"
```
which is much less clear. 
I'll give it more thought :wink:

Reply:
I’m ok with both :)